x-pack/plugins/actions/server/builtin_action_types/servicenow/schema.ts

```diff
@@ -9,6 +9,7 @@ import { schema } from '@kbn/config-schema';
 
 export const ExternalIncidentServiceConfiguration = {
   apiUrl: schema.string(),
+  isLegacy: schema.boolean({ defaultValue: false }),
```

Comment:
Do you think is better if I put it outside of the config schema like the `isMissingSecrets`?

Reply:
cc @mikecote @gmmorris @pmuellr @YulNaumenko

Reply:
Good question Christos.
My _gut_ instinct is to keep it limited to SN for now, and if we encounter another situation where we need to support legacy connectors we can explore a generic approach and a migration path.

But I'll wait for some thoughts from the rest of the team :)

Reply:
Thanks, @gmmorris. I am asking mostly because we gonna make global changes based on this attribute. For example in the UI page with all connectors:

<img width="280" alt="Screenshot 2021-09-27 at 7 16 01 PM" src="https://user-images.githubusercontent.com/7871006/134948110-c5816765-d6cc-4241-afd1-f4a7275db970.png">

Reply:
oh 🤔  I see.
That does actually make sense then.

But a change of that magnitude should happen in a separate PR in my opinion. 
This PR is huge and to make sure a change like that is properly reviewed I'd prefer if we do it as a follow-up.

Is that an option?

Reply:
Of course. It depends on how fast this PR gonna get merged. Although we can do it after FF as a fix.

Reply:
Well, if possible, I'd like to split them apart as because I believe smaller PRs are faster, safer and easier to review... 🤷 

If it isn't **really** a fix, we shouldn't do it after FF, and it would be better if we simply deliver the generalization in 8.0/8.1 if it's merged after FF.
If you're saying this is something we can't easily generalise after the fact then I'd say we should do it in _this_ PR, but in the future it would be better to break this sort of thing down into smaller PRs that are easier and faster to review.
In the long run - that's faster.

Reply:
I leave this up to you, obviously, I'm just trying get us to work in smaller increments as I believe it's more sustainable.

Reply:
Thank! It can be generalized after this PR. We can discuss it offline after FF if we can do it as a fix or deliver it in 8.0/8.1.